docs/cip/README.md

```diff
@@ -461,247 +1988,671 @@ The Leios protocol may have to mitigate the following categories of threats.
-
-*Network interference:* Malicious network activity such as violating the sync protocol or delaying diffusion of block or votes creates a minor annoyance that the node's sync protocol will quickly avoid by preferring efficient and honest nodes. Large numbers of malicious relays would be needed to impinge on efficiency even in a small degree.
-
-*Denial of service:* Transaction-based denial of service attacks on Leios would involve submitting numerous invalid, duplicate, or conflicting transactions to different nodes so that they would all make their way into the memory pool and then to IBs, only to be invalidated when transaction reconciliation occurs after those IBs are indirectly referenced by a certificate on a Praos ranking block. Such a denial of service would result in extra computation by the nodes and wasted permanent storage in the IBs. (Plutus transactions may be especially burdensome in this respect.) Ongoing research will mitigate such denial of service via sharding techniques and Leios's fee structure. Sharding will prevent duplicate transactions from reaching IBs and the fee structure will enforce payment for intentionally conflicted transactions, even though only one of the transactions would make it onto the ledger.
-
-### Resource requirements
-
-The resource requirements for operating Leios nodes have been estimated from benchmarking and simulation studies. The benchmark values for various Leios operations come either from measurements of the cryptography prototype[^3] or from the IOG benchmarking cluster for the Cardano node. These were input to the Haskell and Rust simulators for Leios to make holistic estimates of resource usage of operating nodes.
-
-> [!CAUTION]
-> 
-> The plots below are placeholders. All of the simulations in this section need to be re-run:
-> 
-> - [ ] Final version of the Leios protocol
-> - [ ] Realistic mainnet topology
-> - [ ] Protocol parameters close to the recommended value
-> - [ ] CPU
->     - [ ] Unlimited?
->     - [ ] Six cores?
-> - [ ] Strip the major titles from the diagrams
-> - [ ] Use SVG format
-
-At high throughput, network egress can become a significant cost for nodes hosted on some cloud-computing providers. The violin plots below indicate that at the higher throughput that Leios can support, network egress can reach nearly 2 MB/s.
-
-![Simulation of Leios network egress](images/network.png)
+_Grinding and other threats to Praos:_ Threats to the ranking blocks used by
+Leios are already mitigated by Ouroboros Praos and Genesis. Nevertheless, the
+possibility of _grinding attacks_, as discussed in [CPS-0017][cps-17], will
```

Comment:
We have a  corresponding CIP number (c.f. [repo](https://github.com/cardano-foundation/CIPs/pull/1065) ) but the PR has not beed merged yet if that is useful.